core/src/Search/Search.js

```diff
@@ -60,10 +60,16 @@ class Search extends Component {
           {...omit(this.props, "innerRef")}
           ref={this.props.innerRef}
           type="search"
-          className={styles.Input}
+          className={
+            this.props.roundedSearch ? styles.RoundedSearch : styles.Input
+          }
           onChange={this.onChange}
+          roundedSearch={this.props.roundedSearch || null}
         />
-        <InputIcon onClick={this.onSubmit}>
+        <InputIcon
```

Comment:
Is it possible to pass className={styles.roundedEdge} which comes from the search component less file